README.md

```diff
@@ -97,7 +106,7 @@ output "subscription_name" {
 | subscription_name | Name of the the azure subscription  |
 
 ```sh
-$ terraform init
+$ terraform init --upgrade
 $ terraform plan # Please verify before applying
 $ terraform apply
 # Once terraform is applied successfully, it will create "client_credentials.json" file.
```

Comment:
Can you modify this line. client_credentials.json will not be created right

Reply:
Done